### PR TITLE
Retracts the student notes feature

### PR DESCRIPTION
**Pull request by lamagnifica**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
## [DOC-2734](https://openedx.atlassian.net/browse/DOC-2734)

Retracts the announcement in LMS and analytics for the notes feature, removes mentions of the Doc additions.

### Date Needed

ASAP

### Reviewers
- [x] Subject matter expert: @scottrish 
- [x] Doc team review (sanity check): @catong @srpearce  

FYI: @jaakana 

### Testing
- [x] Ran ./run_tests.sh without warnings or errors

### Post-review
- [x] Add description to release notes task for next week? as a comment
- [ ] Squash commits

**Comment by srpearce** [comment]
:+1: :disappointed:

**Comment by pdesjardins** [comment]
Looks good. :+1:

File: en_us/release_notes/source/2016/analytics/analytics_0222_2016.rst [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 packages. Among these events are ``edx.course.student_notes.added``,
 ``edx.course.student_notes.edited``, and
-``edx.course.student_notes.used_unit_link``. For more information, see
-:ref:`data:notes` in the *Edx Research Guide*.
+``edx.course.student_notes.used_unit_link``.
+
+.. For more information, see :ref:`data:notes` in the *Edx Research Guide*.
+
+**Update 25 Feb 2016** This feature is not yet available for courses running on
+the edx.org or edge.edx.org web sites. Delivery of the notes feature is
```
Comment (catong) [inline en_us/release_notes/source/2016/analytics/analytics_0222_2016.rst]: @lamagnifica want to make case consistent for "Notes feature" vs "notes feature"?

File: en_us/release_notes/source/2016/lms/lms_0222_2016.rst [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 The learner can then review his notes in the body of the course or on a
-separate **Notes** page. For more information, see
-:ref:`partnercoursestaff:Notes Tool` in
-the *Building and Running an edX Course* guide or :ref:`learners:SFD Notes` in the *EdX Learner's Guide*.
+separate **Notes** page.
+
+.. For more information, see :ref:`partnercoursestaff:Notes Tool` in the *Building and Running an edX Course* guide or :ref:`learners:SFD Notes` in the *EdX Learner's Guide*.
+
+**Update 25 Feb 2016** This feature is not yet available for courses running on
+the edx.org or edge.edx.org web sites. Delivery of the notes feature is
```
Comment (catong) [inline en_us/release_notes/source/2016/lms/lms_0222_2016.rst]: "Notes feature" or "notes features" whichever you go with from above.

File: en_us/release_notes/source/2016/analytics/analytics_0222_2016.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/release_notes/source/2016/analytics/analytics_0222_2016.rst]: it should be lc, correct?

**Comment by lamagnifica** [comment]
verbal thumbs up from Scott

**Pull request merged**